lib/logstash/outputs/pravega.rb

```diff
@@ -90,7 +93,7 @@ def create_producer
       logger.debug("created stream successfully", :stream => @stream_name)
```

Comment:
You should close streamManager in this function. It isn't needed after creating the stream.

Reply:
Still need to do this.